ui/src/dashboards/components/DashboardsTable.tsx

```diff
@@ -60,7 +60,12 @@ class DashboardsTable extends PureComponent<Props> {
           {_.sortBy(dashboards, d => d.name.toLowerCase()).map(dashboard => (
             <tr key={dashboard.id}>
               <td>
-                <Link to={`${dashboardLink}/dashboards/${dashboard.id}`}>
+                <Link
+                  to={`${dashboardLink}/dashboards/${dashboard.id}`}
+                  data-test={`${dashboard.name
```

Comment:
Shouldn't this be simply `dashboard.name` so that extra computations are avoided?

Reply:
The structure of the data-test attributes across all the elements is lower case and the hyphen instead of space. Now, if that is not necessary to have, I agree that it can be changed to `dashboard.name`.

Reply:
conventions are good, I only think that it could have been anything simpler, such as `dashboard-link-${dashboard.id}` or `dashboard-link:${dashboard.name}`.

Reply:
Attribute value changed to `dashboard-link-${dashboard.id}`. Test using this element adjusted accordingly.